packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -132,76 +138,66 @@ export class CheckpointService implements TreeDataProvider<BaseNode> {
     return element.getChildren();
   }
 
-  // There's a limit to the number of checkpoints for a given user and
-  // that number is 5.
-  public canAddCheckpointNote(): Boolean {
-    return this.getChildren().length < MAX_ALLOWED_CHECKPOINTS;
+  public hasFiveOrLessActiveCheckpoints(displayError: boolean): Boolean {
+    let numEnabledCheckpoints = 0;
+    for (const cpNode of this.getChildren() as CheckpointNode[]) {
+      if (cpNode.isCheckpointEnabled()) {
+        numEnabledCheckpoints++;
+      }
+    }
+    const fiveOrLess: boolean =
```

Comment:
Boolean type is implicit; you don't need to declare it.

Reply:
This should fail the `npm lint` as well because we turn on https://palantir.github.io/tslint/rules/no-inferrable-types/ on https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/tslint.json#L22